### ISSUE TITLE
put in html partials for header and footer

### ISSUE THREAD

**Issue by louh** [mapzen/styleguide#33]

**Comment by souperneon** [comment]
@louh @hanbyul-here whoever works on this, we can use the _body-small_ class I just created for 14px text size. (you can also rename that class but I wanted to create it coz we use that size in other places as well)

**Comment by louh** [comment]
For building components, we might want to use a variable instead of a class. e.g.

``` css
.my-component {
  font-size: $body-small; /* Or whatever we name it */
}
```

This sets the component's intended font size inside of the component itself. Adding a helper class to adjust font-size will complicate the markup, and it makes it so that anyone who inserts `my-component` into a page will have to remember to add the correct helper classes, which leaves room for error.

We can still expose a helper class if we think people will want to use it for things (see: helper classes for colors), but we shouldn't have to build the components with them, I don't think.

**Comment by souperneon** [comment]
I added $body-small to variables.scss. I removed it from the general styleguide page. We can show it used in a component in the future.

**Comment by louh** [comment]
:+1: just used it to fix the side-nav font size :)

**Comment by souperneon** [comment]
@louh have you already worked on this? should it live in ui-components?

**Comment by migurski** [comment]
CfA examples for comparison:

• https://github.com/codeforamerica/codeforamerica.org/tree/master/fragments
• https://www.codeforamerica.org/fragments/global-header.html
• https://www.codeforamerica.org/fragments/global-footer.html

**Comment by souperneon** [comment]
@louh are you working on this already?

**Comment by louh** [comment]
@souperneon yes

**Comment by souperneon** [comment]
:+1:

**Comment by louh** [comment]
I'm not too picky about the name; `partials` is common for static HTML includes, but I have no objection to `fragments` if that's clearer for people

**Comment by souperneon** [comment]
I'm fine either way as long as we stay consistent. @hanbyul-here @migurski ^^

**Comment by migurski** [comment]
Was this closed in #130?

**Comment by louh** [comment]
Yep!
